src/BlazorWebView/src/Maui/Android/BlazorWebViewHandler.Android.cs

```diff
@@ -99,6 +99,10 @@ private void StartWebViewCoreIfPossible()
 				VirtualView.JSComponents,
 				hostPageRelativePath);
 
+			var args = new BlazorWebViewInitializingEventArgs();
+			((BlazorWebView)VirtualView).NotifyBlazorWebViewInitializing(args);
+			args.OnWebViewInitialized?.Invoke(PlatformView);
```

Comment:
I was about to say that it's sad that for Android we don't give any special settings access. It seems on Android we pre-set the settings "very early" so there's nothing to give here. But reading the Android WebView docs, it seems likely that the user can set the settings directly on the WebView we provide here to update settings after it's been created.

E.g. these properties on the settings all seem to be settable once it's running: https://developer.android.com/reference/android/webkit/WebSettings

But, I haven't tried it. I also can't think of any reasonable alternative if this somehow isn't the case.